routes/ToDoRoutes.ts

```diff
@@ -0,0 +1,22 @@
+import { Router } from "express";
+import {
+  getAllToDos,
+  getToDoByID,
+  postToDo,
+  updateToDo,
+  deleteToDo,
+} from "../api_functions/ToDoAPIFunctions";
+
+const todoRouter = Router();
+
+todoRouter.get("/toDos", (req, res) => getAllToDos(req, res));
```

Comment:
you can directly call the function as a 2nd parameter. instead of calling it in an array function